admin/app/components/solidus_admin/orders/index/component.html.erb

```diff
@@ -9,7 +9,7 @@
       <%= render component("ui/button").new(
         tag: :a,
         text: t('.create_order'),
-        href: spree.new_admin_order_path,
+        href: solidus_admin.new_order_path,
```

Comment:
🚨 Important: This PR cannot be merged as-is because it redirects to an `orders/new` component that isn’t ready for use.